actions/sharedconfig/routes.go

```diff
@@ -19,6 +19,9 @@ func NewHandler(appConfig *config.AppConfig, services *config.AppServices) route
 	sharedConfigEndpoints := routes.APIEndpointsFunc(func(router *gin.RouterGroup) {
 		group := router.Group("/shared-config")
 		group.GET("", action.get)
+
+		group2 := router.Group("/configs/shared")
+		group2.GET("", action.get2)
```

Comment:
Because we need different prefix (`/api/v1`) we should probably create totally new type and new method for returning those routes, or maybe change NewHandler method to return a list instead of single thing.

Reply:
Done